posthog/client.py

```diff
@@ -466,6 +467,11 @@ def capture(
         require("properties", properties, dict)
         require("event", event, string_types)
 
+        # Grab current context tags, if any exist
+        context_tags = get_tags()
+        if context_tags:
+            properties.update(context_tags)
```

Comment:
this placement is cool
since it looks to me like i can set a super property
and then use context to override it sometimes

that is pleasing